### ISSUE TITLE
Can't get this working using yarn

### ISSUE THREAD

**Issue by scottweinert** [Junkern/domrec-puppeteer#1]
I'm getting:
```
Error: ENOENT: no such file or directory, open './node_modules/domrec-puppeteer/node_modules/domrec-core/dist/recording.js'
```
`node_modules/domrec-puppeteer` folder exists, but `node_modules/domrec-puppeteer/node_modules` does not exist. The `domrec-core` package and it's associated `dist`  folder is located in my main `node_modules` folder.

**Comment by scottweinert** [comment]
The issue for me stems from line 18 of `index.js`:

```
const filePath = path.join(__dirname, '..', 'node_modules', 'domrec-core', 'dist', 'recording.js')
```

Changing it to this fixed it for me:

```
    const filePath = path.join(
      __dirname,
      '../..',
      'domrec-core',
      'dist',
      'recording.js'
    )
```

**Comment by Junkern** [comment]
Yeah, working with file paths pointing to `node_modules` is not a good solution, as you have discovered. I think the best option for now is to directly include the `recording.js` inside this package. I will make a PR tomorrow.

**Comment by Junkern** [comment]
Published with version `0.0.2` @scottweinert
